### PR TITLE
Update colors

### PR DESCRIPTION
**Pull request by wenjebs**
The UI is a beautiful blue now!

### CODE REVIEW

File: src/main/resources/view/DarkTheme.css [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .background {
-    -fx-background-color: derive(#1d1d1d, 20%);
-    background-color: #383838; /* Used in the default.html file */
+    -fx-background-color: derive(#4A628A, 20%);
+    background-color: #4A628A; /* Used in the default.html file */
```
Comment (hyxnnii) [inline src/main/resources/view/DarkTheme.css:3]: Demure choice of colour!

**Review by hyxnnii — COMMENTED** [review]
lgtm~

File: src/main/resources/view/DarkTheme.css [hunk 2]
```diff
@@ -217,7 +217,7 @@
     -fx-border-color: #e2e2e2;
     -fx-border-width: 2;
     -fx-background-radius: 0;
-    -fx-background-color: #1d1d1d;
+    -fx-background-color: #4A628A;
     -fx-font-family: "Segoe UI", Helvetica, Arial, sans-serif;
```
Comment (yimjianbing) [inline src/main/resources/view/DarkTheme.css:221]: Maybe can change font in future ui changes to make it more unique!

File: src/main/resources/view/DarkTheme.css [hunk 3]
```diff
@@ -77,20 +77,20 @@
 }
 
 .split-pane:horizontal .split-pane-divider {
-    -fx-background-color: derive(#1d1d1d, 20%);
+    -fx-background-color: derive(#4A628A, 20%);
     -fx-border-color: transparent transparent transparent #4d4d4d;
 }
 
 .split-pane {
     -fx-border-radius: 1;
```
Comment (yimjianbing) [inline src/main/resources/view/DarkTheme.css:85]: Maybe adjust some border radius to make it rounder!

**Review by yimjianbing — APPROVED** [review]
LGTM! nice job

**Pull request merged**